--- a/tests/Feature/DatabaseMigrationTest.php
+++ b/tests/Feature/FeatureDatabaseMigrationTest.php
@@ -1,4 +1,4 @@
 <?php

-namespace Tests\Feature;
+namespace Tests\Feature;

-class DatabaseMigrationTest extends TestCase
+class FeatureDatabaseMigrationTest extends TestCase
 {
--- a/tests/Unit/Deployment/DatabaseMigrationTest.php
+++ b/tests/Unit/Deployment/UnitDatabaseMigrationTest.php
@@ -1,4 +1,4 @@
 <?php

-namespace Tests\Unit\Deployment;
+namespace Tests\Unit\Deployment;

-class DatabaseMigrationTest extends TestCase
+class UnitDatabaseMigrationTest extends TestCase
 {
